org.twinery.Twine.yaml

```diff
@@ -0,0 +1,72 @@
+---
+app-id: "org.twinery.Twine"
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+runtime: "org.freedesktop.Platform"
+runtime-version: "21.08"
+sdk: "org.freedesktop.Sdk"
+sdk-version: "21.08"
+command: "twine-wrapper"
+copy-icon: true
+separate-locales: false
+finish-args:
+  # Required due to being a GUI application
+  - --socket=x11
```

Comment:
Missing `--device=dri`.

Reply:
Uhm, why? I don't see where I would need hardware acceleration in a text editior or does Electron use this by default these days?

Reply:
According to [this](https://github.com/electron/electron/blob/main/docs/tutorial/offscreen-rendering.md), hardware rendering needs to be explicitly disabled in order to enable software rendering, or like what happening here, fail creating the GL context by not having a DRI device.  
With intel_gpu_top, I can see significant change in the render engine usage when the app is run with DRI device available in the sandbox.

Reply:
👍🏻